steely/plugins/beer.py

```diff
@@ -0,0 +1,78 @@
+from steely import config
+import requests
+
+# API stuff
+
+URL = 'http://api.brewerydb.com/v2/'
+SEARCH = URL + "search/"
+API_KEY = config.BREWERYDB_API_KEY
+
+# error stuff
+ERR_NO_RESULTS = "ah now we got no results"
+ERR_API_LIMIT = "we used up all our api juice, try again tomorrow"
+ERR_WHAT = "no idea whats going on"
```

Comment:
is there supposed to be a command mentioned here somewhere

Reply:
fixed it